torch/optim/adam.py

```diff
@@ -105,6 +105,10 @@ class Adam(Optimizer):
         capturable (bool, optional): whether this instance is safe to capture in a CUDA graph.
             Passing True can impair ungraphed performance, so if you don't intend to
             graph capture this instance, leave it False (default: False)
+        differentiable (bool, optional): whether autograd should occur through the optimizer step in training
+            otherwise, the step() function runs in a torch.no_grad() context,
+            Setting to True can impair performance,
+            so leave it False if you don't intend to run autograd through this instance. (default: False) 
```

Comment:
```suggestion
        differentiable (bool, optional): whether autograd should occur through the optimizer step in training,
            otherwise, the step() function runs in a torch.no_grad() context. 
            Setting to True can impair performance, so leave it False if you don't intend to run autograd 
            through this instance. (default: False) 
```
some punctuation nits--thank you for following through!

Reply:
Thank you